src/pytest_html/plugin.py

```diff
@@ -757,3 +767,64 @@ def pytest_sessionfinish(self, session):
 
     def pytest_terminal_summary(self, terminalreporter):
         terminalreporter.write_sep("-", f"generated html file: file://{self.logfile}")
+
+
+class NextGenReport:
```

Comment:
can you move this into a separate file? I want to break apart `plugin.py` so it's easier to read/develop it ( the PR for that is [here](https://github.com/pytest-dev/pytest-html/pull/427) )

Reply:
Good idea!

Can you create a ticket for it in the project? Or I will forget.

Reply:
Sure @BeyondEvil. Added a card in the todo column